1.hello-world/README.md

```diff
@@ -24,7 +24,7 @@ Follow [instructions](https://github.com/dapr/docs/blob/master/getting-started/e
 Now that we've locally set up Dapr, clone the repo, then navigate to the Hello World sample: 
 
 ```bash
-git clone https://github.com/dapr/samples.git
```

Comment:
It'd be nice if we can point to the supported versions table on landing page. We should also take into account people using `edge` version of dapr runtime. So, can we please add the instructions like below:

```bash 
git clone -b [<dapr_version_tag>](https://github.com/dapr/samples#supported-dapr-runtime-version) https://github.com/dapr/samples.git
```
> Note: Ignore checking out the tag if using `edge` version of dapr runtime. Use `git clone https://github.com/dapr/samples.git` instead.

```bash
cd samples/1.hello-world
```

Reply:
Thanks for updating. I have merged master into this branch and there is a new sample in this branch now. So please do the same change for 11.hello-dapr-slim as well.

Reply:
This is hopefully ok now.

Br

Hans K.